src/components/VersionChooser/index.tsx

```diff
@@ -43,42 +50,83 @@ type PropsFromState = {
   versions: VersionsState;
 };
 
-export type PropsFromRouter = {
-  lang: string;
-};
-
-type RouterProps = RouteComponentProps<PropsFromRouter>;
+type RouterProps = RouteComponentProps<{}>;
 
 type Props = PublicProps &
   ConnectedReduxProps &
   DefaultProps &
   PropsFromState &
   RouterProps;
 
-export class VersionChooserBase extends React.Component<Props> {
+type State = {
+  baseVersionId: number | undefined;
+  headVersionId: number | undefined;
+};
+
+export class VersionChooserBase extends React.Component<Props, State> {
   static defaultProps: DefaultProps = {
     _fetchVersionsList: fetchVersionsList,
     _higherVersionsThan: higherVersionsThan,
     _lowerVersionsThan: lowerVersionsThan,
   };
 
+  constructor(props: Props) {
+    super(props);
+    this.state = { baseVersionId: undefined, headVersionId: undefined };
+  }
+
   componentDidMount() {
     const { _fetchVersionsList, addonId, dispatch, versionsMap } = this.props;
 
     if (!versionsMap) {
       dispatch(_fetchVersionsList({ addonId }));
     }
+
+    this.initializeBaseVersion();
+  }
+
+  componentDidUpdate() {
+    this.initializeBaseVersion();
+  }
+
+  initializeBaseVersion() {
```

Comment:
Not exactly. Some views, like the home page, have neither a head version or base version. The head version chooser doesn't need to be initialized because it's default value is its highest one (what we want). The initializer makes sure the default value of the base version chooser is its lowest one. 

However, none of this matters when neither a base version or head version exists because the popover / chooser is not displayed for that case.